src/eventloop.jl

```diff
@@ -18,7 +18,15 @@ function eventloop(socket)
                 end
             finally
                 @async begin
-                    sleep(idle_delay[])
+                    if check_interact_loaded() && idle_delay[] < interact_min_delay
```

Comment:
I'm alright with that, I can update my branch to eliminate the idle delay functionality.